services/rules.js

```diff
@@ -476,7 +476,7 @@ async function applyRecursiveRules(_rules, query) {
 }
 
 // eslint-disable-next-line no-unused-vars
-const execRules = async (owner_type, products = []) => {
+const execRules = async (owner_type, products = [], option) => {
```

Comment:
option should have a default parameters of `undefined` or `null`
option should be rename because it's seem it take an id, e.g: `ownerId`